### PR TITLE
UXD-309 refactor to CSF (Checkbox, Confirmation, Switch)

### PR DESCRIPTION
**Pull request by slee1130**
### Purpose 🚀

Stories needs to be refactored with the CSF

### Notes ✏️

CSF Stories
Refactor the stories for a component to use CSF instead of storiesOf syntax.
Reference: Component Story Format (CSF)

- Checkbox
- Confirmation
- Switch

### Updates 📦
If you have changed a component's source code (not stories, specs, or docs), before merging your branch run `yarn changeset`. This will prompt you to:
- indicate if changes were patch/minor/major for each modified package
- enter a release message

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/xd-309-RefactorCSF2

### Screenshots 📸
_optional but highly recommended_

### References 🔗

https://aclgrc.atlassian.net/browse/UXD-309

### CODE REVIEW

File: .changeset/few-badgers-mix.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@paprika/checkbox": patch
+"@paprika/confirmation": patch
+"@paprika/switch": patch
+---
+
+Refactor to component story format
```
Comment (mikrotron) [inline .changeset/few-badgers-mix.md:7]: Don't need the `changeset` for this PR

File: packages/Checkbox/stories/Checkbox.stories.js [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
+import ExampleStory from "storybook/components/ExampleStory";
 import Variations from "./examples/Variations";
-import CheckboxGrouping from "./examples/CheckboxGrouping";
 import ShowcaseStory from "./examples/Showcase";
+import Checkbox from "../src";
 
-const storyName = getStoryName("Checkbox");
+export default {
+  title: getStoryName("Checkbox"),
+  component: Checkbox,
```
Comment (mikrotron) [inline packages/Checkbox/stories/Checkbox.stories.js:12]: Maybe don't need this

File: packages/Confirmation/stories/Confirmation.examples.stories.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+const storyName = getStoryName("Confirmation");
+
+const confirmationStoryParameters = {
+  parameters: {
+    viewMode: "story",
+    options: {
+      isToolshown: false,
+      docs: { disable: true },
+    },
+  },
```
Comment (mikrotron) [inline packages/Confirmation/stories/Confirmation.examples.stories.js:22]: I think you could use `exampleStoryParameters` instead from

`import { exampleStoryParameters } from "storybook/assets/storyParameters";`

Comment (mikrotron) [reply packages/Confirmation/stories/Confirmation.examples.stories.js:22]: Unless `docs: { disable: true },` does something important?

File: packages/Confirmation/stories/examples/Showcase.js [hunk 4]
```diff
@@ -27,21 +26,16 @@ const confirmationProps = () => ({
   isPending: boolean("Is Pending", false),
 });
 
-const ExampleStory = props => (
+const ConfirmationStory = props => (
   <>
```
Comment (mikrotron) [inline packages/Confirmation/stories/examples/Showcase.js:30]: This fragment isn't necessary.

File: packages/Confirmation/stories/Confirmation.stories.js [hunk 5]
```diff
@@ -1,61 +1,16 @@
...
-    </Story>
-  ))
-  .add("ConfirmationExample with async action", () => (
-    <Story>
-      <ConfirmationExampleWithAsyncAction />
-    </Story>
-  ));
-
-storiesOf(`${storyName}/Backyard/Tests`, module).add("Screener", () => <ConfirmationExample />);
+export const showcase = ShowcaseStory;
```
Comment (mikrotron) [inline packages/Confirmation/stories/Confirmation.stories.js:12]: You _could_ wrap the `<ExampleStory>` from `Showcase.js` around the `<ShowcaseStory>` here... but that's not actually in scope of this task – which is specifically just to refactor to CSF (and that you've done a nice job of 🙂)

File: packages/Switch/stories/Switch.stories.js [hunk 6]
```diff
@@ -1,32 +1,34 @@
...
-      <Example
-        a11yText={text("a11yText", "")}
-        canPropagate={boolean("canPropagate", true)}
-        isDisabled={boolean("isDisabled", false)}
-        size={select("size", [types.SMALL, types.MEDIUM, types.LARGE], "medium")}
-      />
-    </Story>
-  ))
-  .add("Variations", () => <Variations />);
+export const showcase = Showcase;
```
Comment (mikrotron) [inline packages/Switch/stories/Switch.stories.js:15]: Same as https://github.com/acl-services/paprika/pull/1216/files#r780579346

File: packages/Checkbox/stories/Checkbox.stories.js (hunk 2 above)
Comment (slee1130) [reply packages/Checkbox/stories/Checkbox.stories.js:12]: When I removed this part, the showcase is gone from the browser.. so probably we need this?

**Review by mikrotron — APPROVED** [review]
llgtm 60

**Pull request merged**